### PR TITLE
PLAT-64196: Fix unexpected scrolling by 5-way key from inside an ExpandableItem

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
When `ExpandableItem` has a `true` value of `lockBottom` prop and `down` key is pressed on the last item of `ExpandableItem`, a scroller which contains `ExpandableItem` scrolls unexpectedly.
Fixed not to make a scrolling in this case. Since `lockBottom` is `true`, there will be no action.

Edit:
Thanks to @MikyungKim, I found that the issue also happens on `up` key when `ExpandableItem` is located on the top of a viewport of a scroller (regardless `lockBottom` is set or not).
Added the fix for this, too.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
A scrolling is a default behavior of web engine by `up` or `down` key. So added `preventDefault()`.

### Links
[//]: # (Related issues, references)
PLAT-64196

### CODE REVIEW

File: packages/moonstone/ExpandableItem/ExpandableItem.js [hunk 1]
```diff
@@ -305,6 +305,7 @@ const ExpandableItemBase = kind({
 				} else if (isDown(keyCode) && wouldDirectionLeaveContainer('down', target)) {
 					if (lockBottom) {
 						ev.nativeEvent.stopImmediatePropagation();
+						ev.nativeEvent.preventDefault();
```
Comment (MikyungKim) [inline packages/moonstone/ExpandableItem/ExpandableItem.js]: ev.preventDefault?

File: packages/moonstone/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/ExpandableItem` to not make scrolling of a parent scroller when `down` key is pressed on the last item
```
Comment (MikyungKim) [inline packages/moonstone/CHANGELOG.md]: Wrong position

File: packages/moonstone/ExpandableItem/ExpandableItem.js [hunk 3]
```diff
@@ -305,6 +305,7 @@ const ExpandableItemBase = kind({
 				} else if (isDown(keyCode) && wouldDirectionLeaveContainer('down', target)) {
 					if (lockBottom) {
 						ev.nativeEvent.stopImmediatePropagation();
+						ev.preventDefault();
```
Comment (MikyungKim) [inline packages/moonstone/ExpandableItem/ExpandableItem.js]: do we need this for 'up' key too?

Comment (0x64) [reply packages/moonstone/ExpandableItem/ExpandableItem.js]: Good point! I just updated to fix `up` key too. The issue happens `ExpandableItem` is located on the top of a viewport of a scroller and the currently focused item is the first one.

**Comment by MikyungKim** [comment]
:ok:

**Pull request merged**